### ISSUE TITLE
Using transition name

### ISSUE THREAD

**Issue by samuveth** [ycs77/headlessui-float#15]
It would be great if it was possible to use transition name instead of having to repeat all my transition styling each time I use Float.

Example of how I usually do it in Vue: 

```
<Transition name="fade">

// Fade transition for Vue's transition and transition-group
.fade-enter-active,
.fade-leave-active {
  transition: opacity 0.2s ease;
}

.fade-enter-from,
.fade-leave-to {
  opacity: 0;
}

```

**Comment by ycs77** [comment]
Released in [v0.8.0](https://github.com/ycs77/headlessui-float/releases/tag/v0.8.0)

### Usage

Use named transition classes to define transition styles:

```html
<Float transition-name="fade">
```

Then you can be add the classes:

```vue
<style>
.fade-enter-active,
.fade-leave-active {
  transition: opacity 0.5s ease;
}

.fade-enter-from,
.fade-leave-to {
  opacity: 0;
}
</style>
```

**Comment by samuveth** [comment]
@ycs77 that was fast :)
